### ISSUE TITLE
Cannot build VS2015 project with grovepi nuget reference

### ISSUE THREAD

**Issue by ser1us** [DexterInd/GrovePi#175]
Hello!

Installed the GrovePi nuget package into my c# windows iot backgroud task template. But I always get the following error:

```
APPX0702: Payload file [USER-Folder]\.nuget\packages\GrovePi\1.0.6\lib\uap10.0\GrovePi\Properties\client.winU.rd.xml' does not exist.
```

Anyone knows what the problem could be?

Best regards,
Christoph

**Comment by penfold** [comment]
Having the same issue. Any further info available?

**Comment by chumiporky** [comment]
I have the same issues, I will comment on the repository of the creator of the nugget package, robsonj

**Comment by dougseven** [comment]
Trying to blink an LED and get this when deploying to the Pi: 

`Payload file 'C:\Users\<username>\.nuget\packages\GrovePi\1.0.6\lib\uap10.0\GrovePi\Properties\client.winU.rd.xml' does not exist.`

Has there been any resolution to this?

**Comment by maxpg** [comment]
Same for me. I downloaded the project and added it directly as a project to my solution.

**Comment by lanselambor** [comment]
Copy 
[ GrovePi/Software/CSharp/GrovePi/Properties/ ]
to 
[ C:\Users\<Your computer name>\.nuget\packages\GrovePi\1.0.6\lib\uap10.0 ]

solved the problem.

**Comment by danvy** [comment]
Issue is still opened. Could you update the NuGet package?

**Comment by rpedersen** [comment]
I have fixed this issue with PR #228.

**Comment by DexterInd** [comment]
Just accepted @rpedersen pull request on this issue.
